src/wrapper/errors.rs

```diff
@@ -39,6 +39,8 @@ pub enum Error {
     ParseFailed(#[source] combine::error::StringStreamError, String),
     #[error("JNI call failed")]
     JniCall(#[source] JniError),
+    #[error("The class of the parameter passed in is invalid.")]
+    InvalidParameterClass
```

Comment:
It seems like it might be reasonable to use `JniError::InvalidArguments`?

I.e. 
```rust
JniCall(JniError::InvalidArguments)
```

The crash/segv/abort happens within a JNI function due to it being passed an invalid argument.

Reply:
Oh that is a good point actually. Missed that one!